posthog/settings/access.py

```diff
@@ -25,6 +25,10 @@
 if get_from_env("DISABLE_SECURE_SSL_REDIRECT", False, type_cast=str_to_bool):
     SECURE_SSL_REDIRECT = False
 
+CSRF_TRUSTED_ORIGINS = [
+    "https://*.posthog.com",
+    "https://*.posthog.dev",
```

Comment:
Setting this was required to make some requests work in the preview environment.

What could we set this to in general? Is there some hostname already flying around?

Reply:
<img width="596" alt="Screenshot 2023-12-08 at 13 08 12" src="https://github.com/PostHog/posthog/assets/984817/fef1a853-d56c-40f7-9d53-cb5ce1d35676">

Ah, missed that scanning the files 🤦 

https://github.com/PostHog/posthog/blob/master/posthog/settings/__init__.py#L56

Reply:
That would fit well, will try.